examples/README.rst

```diff
@@ -0,0 +1,8 @@
+.. _gallery_examples:
+
+Examples
+========
+
+A gallery of interactive examples that showcase how the tools provided ``neuraloperator`` can be applied to a variety of problems. 
+Check out :ref:`_user_guide` for more detailed information on the theory behind neural operators. 
+
```

Comment:
Why did you rename it back to README from the newer GALLERY_HEADER?

Reply:
Might be a sphinx versioning issue